stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetAddCardFragment.kt

```diff
@@ -56,23 +53,4 @@ internal class PaymentSheetAddCardFragment(
             }
         }
     }
-
-    override fun createHeaderText(
-        config: FragmentConfig
-    ): String {
-        val amount = config.paymentIntent.amount
-        val currencyCode = config.paymentIntent.currency
-        return if (amount != null && currencyCode != null) {
-            val currency = Currency.getInstance(
-                currencyCode.toUpperCase(Locale.ROOT)
-            )
-
-            resources.getString(
-                R.string.stripe_paymentsheet_pay_using_with_amount,
-                CurrencyFormatter().format(amount, currency)
-            )
-        } else {
-            resources.getString(R.string.stripe_paymentsheet_pay_using)
-        }
-    }
```

Comment:
Why are we removing this?

Reply:
To match the iOS version. The amount is not shown on the header, only on the button.